### ISSUE TITLE
2.0.0-beta.3 no working in IE

### ISSUE THREAD

**Issue by nolafs** [bfwg/ngx-drag-scroll#165]
Hi 

2.0.0-beta.3 doesn't work in IE. The AOT build throws in error in the browsers. I tried version 1.8 but this gives me error in Angular 6.

Any ideas?

Cheers
Olaf

**Comment by bfwg** [comment]
Thanks for reporting, I'll take a look when I have the chance.

**Comment by bogeorge** [comment]
I'm seeing the same think in beta4.  I think it's because the class keyword isn't transpiled

**Comment by menahemc** [comment]
there is any way to make it works on IE11? 
did you fixed this issue yet?
please.... i need immediate solution for this issue.
Thanks

**Comment by nolafs** [comment]
Hey,

Is downloaded it and added the module to my project. That fixed it.

Cheers
Olaf

Sent from my iPhone

On 19 Aug 2018, at 13:53, menahemc <notifications@github.com<mailto:notifications@github.com>> wrote:


there is any way to make it works on IE11?
did you fixed this issue yet?
please.... i need immediate solution for this issue.
Thanks

**Comment by nolafs** [comment]
Did u amended the polyfill?


Sent from my iPhone

On 19 Aug 2018, at 15:19, menahemc <notifications@github.com<mailto:notifications@github.com>> wrote:


i downloaded it but steel i get the same issue on main.js

**Comment by menahemc** [comment]
yes, if i remove the drag-scroll from my project everithing work great. my err is 
'SCRIPT1002: Syntax error' on position of:

class DragScrollItemDirective {
    constructor(elementRef) {
        this.elementRef = elementRef;
        this.display = 'inline-block';
        this._elementRef = elementRef;
    }
}'

**Comment by michalco16** [comment]
hey...
I'm also dealing the same problem as you.. :(
IE not working 
SCRIPT1010: Expected identifier..polyfills
I comment out all the requires polyfills in the polyfills.ts
I run the following
1-npm install --save classlist.js
2-npm install --save web-animations-js
still not working 
what can I do ?

**Comment by menahemc** [comment]
@nolafs i downloaded it in zip and i open it as a stand alone project and it works, I guess that there is issue with the ts translation. but how do i attach the module to my project locally??
Thanks

**Comment by nolafs** [comment]
Just copy the the module into folder of your project and import them like u would do with any other modules.

Sent from my iPhone

On 20 Aug 2018, at 11:34, menahemc <notifications@github.com<mailto:notifications@github.com>> wrote:


@nolafs<https://github.com/nolafs> i downloaded it in zip and i open it as a stand alone project and it works, I guess that there is issue with the ts translation. but how do i attach the module to my project locally??
Thanks

**Comment by menahemc** [comment]
It works:) 
ThankU so mutch

**Comment by bfwg** [comment]
Sorry for the late response guys. I think I know what the issue is. The build target in `tsconfig` file is pointing to `es2015` instead of `es5`. I'll try to make a PR today and test it out.

https://github.com/bfwg/ngx-drag-scroll/blob/develop/src/tsconfig.json#L4

**Comment by bfwg** [comment]
The PR is up. @nolafs and @menahemc , can you install the experiment branch and give it a test?
You can test the branch by putting the below in package.json and re-install the node_modules(remove it first, and run `npm install`).
```
    "ngx-drag-scroll": "git+ssh://git@github.com/bfwg/ngx-drag-scroll.git#not-working-in-ie-11",

```

**Comment by menahemc** [comment]
i try it, i got this:
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t ssh://git@github.com/bfwg/ngx-drag-scroll.git
npm ERR!
npm ERR! ssh: connect to host github.com port 22: Connection timed out
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR! exited with error code: 128
npm ERR! A complete log of this run can be found in:

**Comment by bfwg** [comment]
My mistake, can you please try the below?
```
    "ngx-drag-scroll": "bfwg/ngx-drag-scroll#not-working-in-ie-11",

```

**Comment by menahemc** [comment]
it dosent works' I got some errors on npm I.

‫בתאריך יום ג׳, 21 באוג׳ 2018 ב-19:19 מאת ‪Fan Jin‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by bfwg** [comment]
I've published `2.0.0-beta.5` which builds the library in ES5( no class keyword). Please give that a try, and let me know the result.

**Comment by bfwg** [comment]
I'm going to assume the issue resolved. Please let me know if I need to reopen this.
